package.json

```diff
@@ -19,6 +19,7 @@
     "test": "cd docs/test; bash test.sh"
   },
   "dependencies": {
+    "glob": "^7.1.1",
```

Comment:
Can we add this tool to the `"bin"` key just a few lines above?  That will make it much easier to use in projects that import this npm package.